src/platform_impl/macos/view.rs

```diff
@@ -199,25 +199,34 @@ declare_class!(
         #[sel(viewDidMoveToWindow)]
         fn view_did_move_to_window(&mut self) {
             trace_scope!("viewDidMoveToWindow");
-            if let Some(tracking_rect) = self.state.tracking_rect.take() {
-                self.removeTrackingRect(tracking_rect);
+            if let Some(tracking_area) = self.state.tracking_area.take() {
+                self.removeTrackingArea(&tracking_area);
             }
 
             let rect = self.visibleRect();
-            let tracking_rect = self.add_tracking_rect(rect, false);
-            self.state.tracking_rect = Some(tracking_rect);
+            let tracking_area = self.init_and_add_tracking_area(
+                NSTrackingAreaOptions::NSTrackingMouseEnteredAndExited
+                    | NSTrackingAreaOptions::NSTrackingMouseMoved
+                    | NSTrackingAreaOptions::NSTrackingActiveInActiveApp,
+                rect,
+            );
+            self.state.tracking_area = Some(tracking_area);
         }
 
         #[sel(frameDidChange:)]
         fn frame_did_change(&mut self, _event: &NSEvent) {
             trace_scope!("frameDidChange:");
-            if let Some(tracking_rect) = self.state.tracking_rect.take() {
-                self.removeTrackingRect(tracking_rect);
+            if let Some(tracking_area) = self.state.tracking_area.take() {
+                self.removeTrackingArea(&tracking_area);
             }
-
             let rect = self.visibleRect();
-            let tracking_rect = self.add_tracking_rect(rect, false);
-            self.state.tracking_rect = Some(tracking_rect);
+            let tracking_area = self.init_and_add_tracking_area(
+                NSTrackingAreaOptions::NSTrackingMouseEnteredAndExited
+                    | NSTrackingAreaOptions::NSTrackingMouseMoved
+                    | NSTrackingAreaOptions::NSTrackingActiveAlways,
+                rect,
+            );
+            self.state.tracking_area = Some(tracking_area);
```

Comment:
Hmm, it sounds like in the documentation that we should instead do this in [`-updateTrackingAreas`](https://developer.apple.com/documentation/appkit/nsview/1483719-updatetrackingareas)?